build/makefiles/actor.mk

```diff
@@ -0,0 +1,112 @@
+# common makefile rules for building actors
+#
+# Before including this, your project Makefile should define the following:
+#
+# Required
+# -----------
+# PROJECT      - Short name for the project, must be valid filename chars, no spaces
+# CLAIMS       - Space-separtaed list of capability contracts to use for signing
+#                These should match the capability providers the actor needs to use.
+#                For example: 
+#                    wasmcloud:httpserver wasmcloud:builtin:logging
+# VERSION      - The actor version number, usually semver format, X.Y.Z
+# REVISION     - A number that should be incremented with every build,
+#                whether or not VERSION has changed
+# REG_URL      - Registry url, e.g. 'localhost;5000' or 'wasmcloud.azurecr.io'
```

Comment:
This semicolon `localhost;5000` should be a colon, `localhost:5000`

Reply:
fixed